devel/undocumented_fields/undocumented_fields.txt

```diff
@@ -0,0 +1,616 @@
+  Class: FluentConnection
+    - connection_properties
+    - finalizer_cbs
+    - health_check
+  Class: FluentConnectionProperties
+    - cortex_host
+    - cortex_pid
+    - cortex_pwd
+    - fluent_host_pid
+    - inside_container
+    - ip
+    - password
+    - port
+  Class: MonitorThread
+    - cbs
+  Class: _ConnectionInterface
+    - scheme_eval
+
+  Module: ansys.fluent.core.launcher.container_launcher
+  Class: DockerLauncher
+    - file_transfer_service
+
+  Module: ansys.fluent.core.launcher.pim_launcher
+  Class: PIMLauncher
+    - file_transfer_service
+
+  Module: ansys.fluent.core.launcher.pyfluent_enums
+  Class: Dimension
+    - THREE
+    - TWO
+  Class: FluentLinuxGraphicsDriver
+    - AUTO
+    - NULL
+    - OPENGL
+    - OPENGL2
+    - X11
+  Class: FluentMode
+    - MESHING
+    - PRE_POST
+    - PURE_MESHING
+    - SOLVER
+    - SOLVER_AERO
+    - SOLVER_ICING
+  Class: FluentWindowsGraphicsDriver
+    - AUTO
+    - DX11
+    - MSW
+    - NULL
+    - OPENGL
+    - OPENGL2
+  Class: LaunchMode
+    - CONTAINER
+    - PIM
+    - SLURM
+    - STANDALONE
+  Class: Precision
+    - DOUBLE
+    - SINGLE
+  Class: UIMode
+    - GUI
+    - HIDDEN_GUI
+    - NO_GRAPHICS
+    - NO_GUI
+    - NO_GUI_OR_GRAPHICS
+
+  Module: ansys.fluent.core.launcher.slurm_launcher
+  Class: SlurmLauncher
+    - file_transfer_service
+
+  Module: ansys.fluent.core.launcher.standalone_launcher
+  Class: StandaloneLauncher
+    - file_transfer_service
+
+  Module: ansys.fluent.core.meshing.meshing_workflow
+  Class: WorkflowMode
+    - FAULT_TOLERANT_MESHING_MODE
+    - TOPOLOGY_BASED_MESHING_MODE
+    - TWO_DIMENSIONAL_MESHING_MODE
+    - WATERTIGHT_MESHING_MODE
+
+  Module: ansys.fluent.core.parametric
+  Class: LocalDesignPoint
+    - name
+  Class: LocalParametricStudy
+    - case_filepath
+    - design_point_table
+
+  Module: ansys.fluent.core.post_objects.meta
+  Class: Attribute
+    - VALID_NAMES
+    - function
+    - name
+  Class: Command
+    - arguments_attrs
+    - command_cls
+    - obj
+  Class: PyLocalContainer
+    - type
+
+  Module: ansys.fluent.core.post_objects.post_helper
+  Class: PostAPIHelper
+    - field_data
+    - field_info
+    - id
+    - obj
+    - surface_api
+
+  Module: ansys.fluent.core.post_objects.post_object_definitions
+  Class: ContourDefn
+    - PLURAL
+  Class: MeshDefn
+    - PLURAL
+  Class: MonitorDefn
+    - PLURAL
+  Class: PathlinesDefn
+    - PLURAL
+  Class: SurfaceDefn
+    - PLURAL
+  Class: Vector
+    - x
+    - y
+    - z
+  Class: VectorDefn
+    - PLURAL
+  Class: XYPlotDefn
+    - PLURAL
+
+  Module: ansys.fluent.core.post_objects.post_objects_container
+  Class: Container
+    - session
+
+  Module: ansys.fluent.core.services.app_utilities
+  Class: AppUtilities
+    - service
+  Class: AppUtilitiesOld
+    - scheme_eval
+
+  Module: ansys.fluent.core.services.batch_ops
+  Class: BatchOps
+    - batching
+
+  Module: ansys.fluent.core.services.datamodel_se
+  Class: Attribute
```

Comment:
It seems it is picking up attributes from Enums as well. Anyways, you're right. We can decide and work on this. We can ignore Enums as well I think.

Reply:
Thanks for pointing it out, I'll skip the enums.

Reply:
Enums are skipped now.